v1/nix/pkgs/fetchPip/fetchPip.nix

```diff
@@ -96,7 +96,11 @@
   # we use mitmproxy to filter the pypi responses
   pythonWithMitmproxy =
     python3.withPackages
-    (ps: [ps.mitmproxy ps.python-dateutil ps.pkginfo ps.packaging]);
+    (ps: [ps.mitmproxy ps.dateutil]);
+
+  pythonWithPackaging =
```

Comment:
Maybe add a comment here on why both python environments are needed.
I tend to forget and already cycled through removing one and reverting it again before.

Reply:
Is this good enough? There are additional comments at the top of the file :)